doc/articles/features/using-skia-gtk.md

```diff
@@ -57,9 +57,55 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 
 ```xml
 <ItemGroup>
-   <PackagReference Include="SkiaSharp" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" />
-   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />
+   <PackagReference Include="SkiaSharp" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
+   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.3" />
 </ItemGroup>
 ```
+
+### .NET Native AOT support
+
+Building an Uno Platform Skia+GTK app with .NET (7+) Native AOT requires, GtkSharp 3.24.24.38 (or later), or Uno Platform 4.7 (or later).
+
+To build an app with this feature enabled:
+1. Add the following property in your `.csproj`:
+   ```xml
+   <PropertyGroup>
+   		<PublishAot>true</PublishAot>
+	</PropertyGroup>
+   ```
+1. Upgrade your project to net7.0:
+   ```xml
+   <TargetFramework>net7.0</TargetFramework>
+   ```
+1. Add the following items in your `.csproj`:
+   ```xml
+   <ItemGroup>
+		<RdXmlFile Include="MyApp.rd.xml" />
+		<RuntimeHostConfigurationOption Include="Switch.System.Reflection.Assembly.SimulatedCallingAssembly" Value="true" />
+   </ItemGroup>
+   ```
+1. Create a file name `MyApp.rd.xml`, and place it next to your csproj:
+   ```xml
+   <Directives>
+      <Application>
+         <Assembly Name="MyApp.Skia.Gtk" Dynamic="Required All" />
+         
+         <Assembly Name="GdkSharp" Dynamic="Required All">
+            <Type Name="Gdk.Screen" Dynamic="Required All"/>
```

Comment:
Does this do anything? If the entire assembly is `Dynamic=Required All`, all types in it are rooted. It shouldn't be necessary to call out individual (non-generic) types.

Reply:
This would mean that removing the `Dynamic="Required All"` from line 95 would be enough. I'll check. Thanks!

Reply:
It indeed does not work when removing both assembly level `Required All`. We'll need to validate further which types can be removed, but at this point, the current will do.